news/lit-findings.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: blog
+title:  Literature Findings for Timeline Research
+date:   2024-07-03 15:44:50 -0400
+permalink: /news/lit-findings
+---
+
+#### by [Yumna Ahmed](/people#yumna-ahmed-25-github)
+
+This summer, I am investigating the effect of timeline orientation with right-to-left language speakers like Urdu, Arabic, Hebrew, and Farsi. Timelines are conventionally written in left-to-right orientation, and little research has been done on the effectiveness of other orientations, *specifically with right-to-left language speakers*. We question whether having timelines in right-to-left orientation for speakers of right-to-left languages results in participants performing better on simple tasks using these timelines. We hope to answer this question in our upcoming article about our study.
+
+To start, we needed to review current literature to analyze how temporal concepts are ordered in these languages. While we conducted our research study in English, Urdu, and Arabic, my focus was primarily on Urdu resources as Urdu is my native language and my preferred language of use at home and school (in Pakistan). 
+
+While completing my primary and secondary education in Pakistan, I was exposed to Urdu literature throughout my life since it is the national language of Pakistan. Interestingly, I do not recall ever seeing a timeline being used in the content I have had exposure to. When this research project was pitched, I found myself thinking deeply about my experiences as I grew up in Pakistan. Having studied in the British CAIE (Cambridge Assessment International Education) system, subjects like history and geography where one may most expect to see timelines were taught in English, while Urdu grammar and literature were taught in Urdu. Islamiyat could be taken in Urdu, but my institution opted for using English.
+
+I studied Urdu and Pakistan Studies (taught in English) as part of my O-Levels (grades 9 through 11). I recall seeing timelines of Pakistan and world history in my *English* history textbooks, or at least summaries of time events before the chapter dove into deeper details of each event. As such, looking into Urdu history textbooks was our first instinct. Did Urdu books use any timelines, now that I was actively on the lookout for them? If so, how did they order the events? Do they choose the conventional left (past) - right (future) order similar (and parallel to) English? Or do the authors write timelines in Urdu that are parallel to Urdu’s right-left read and write direction? 
+
+I found all sorts of resources  - from old scriptures to modern textbooks taught in schools today. Among those, a select few that stood out:
+ * *'Mashrati Uloom'* for Grade 5 [link here][fifth-grade-sst]
+ * Monthly *'Phool'* Digest September 2014 [link here][flower-mag]
```

Comment:
This link isn't working and I'm not familiar with the syntax you're using for the links. Usually, I have [my text](https://my-url.com) in parentheses instead of brackets. The first link works but not the second or third. @yuahmed

Reply:
I think i fixed it now - the links had brackets around them which were causing the issue I believe. I followed the syntax as found in template.md